src/tools/cephfs/JournalScanner.h

```diff
@@ -97,8 +105,9 @@ class JournalScanner
   typedef std::map<uint64_t, EventRecord> EventMap;
   typedef std::map<uint64_t, EventError> ErrorMap;
   typedef std::pair<uint64_t, uint64_t> Range;
-  bool pointer_present;
-  bool pointer_valid;
+  bool is_mdlog;
```

Comment:
How about a `enum type` for the kind of journal?

Reply:
there are codes using Journaler pointer to get the journal inode number,  since Journaler pointer is only for "mdlog".  so, I use is_mdlog to quickly identify if it's "mdlog", no need to know the kind of journal. and i don't want to use string comparing "type_ == mdlog" everywhere, that's why i introduced an extra bool variable here.